### PR TITLE
qemu 2.8.0

### PR DESCRIPTION
**Pull request by JCount**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----
also fix lazy symbol binding on 10.11

replaces #8089

### CODE REVIEW

File: Formula/qemu.rb [hunk 1]
```diff
@@ -42,6 +42,14 @@ class Qemu < Formula
   def install
     ENV["LIBTOOL"] = "glibtool"
 
+    # Fixes "dyld: lazy symbol binding failed: Symbol not found: _clock_gettime"
```
Comment (ilovezfs) [inline Formula/qemu.rb]: Do we know if there's an upstream report for this yet?

**Comment by EugenMayer** [comment]
waiting for this to happen. Thanks!

File: Formula/qemu.rb (hunk 1 above)
Comment (JCount) [inline Formula/qemu.rb]: I could not find one.

**Review by ilovezfs — APPROVED** [review]

**Comment by frranck** [comment]
Hi @JCount
Is there a reason why SDL is disabled ?
--sdl
qemu-system-ppc: --sdl: SDL support is disabled

http://braumeister.org/formula/qemu says sdl is required...

**Comment by JCount** [comment]
@frranck by default `cocoa` is used for the user interface. Building `--with-sdl` (and/or `--with-gtk+`) disables the cocoa UI and enables the specified libraries instead. http://braumeister.org/formula/qemu seems to be out of date. Also there seems to be an intermitten `x11` related bug currently when trying to build `--with-sdl`. #4707

**Comment by frranck** [comment]
@JCount  If I run "retroarch" alfter login in the qemu-system-ppc window, I just get a blank screen.
How do you get graphical things to work ?

**Comment by JCount** [comment]
@frranck I suggest you look at the upstream documentation on how to use `qemu` and, if you think you have found a bug or need further assistance, please contact upstream. A place to start might be http://wiki.qemu.org/Contribute/MailingLists.

**Pull request closed without merging**